docs/book/drafts/spaces-sdks-docker-zenml.md

```diff
@@ -0,0 +1,157 @@
+# ZenML on Spaces
+
+[ZenML](https://github.com/zenml-io/zenml) is an extensible, open-source MLOps framework for creating portable, production-ready MLOps pipelines. It's built for Data Scientists, ML Engineers, and MLOps Developers to collaborate as they develop to production.
+
+ZenML offers a simple and flexible syntax, is cloud- and tool-agnostic, and has
+interfaces/abstractions catered toward ML workflows. With ZenML you'll have all
+your favorite tools in one place, so you can tailor a workflow that caters to
+your specific needs.
+
+This Huggingface Space allows you to get up and running with a deployed version
+of ZenML with just a few clicks. Within a few minutes, you'll have this default
+ZenML dashboard deployed and ready for you to connect to from your local
+machine.
+
+In the sections that follow, you'll learn to deploy your own instance of ZenML and use
+it to view and manage your machine learning pipelines right from the Hub. ZenML
+on Huggingface Spaces is a **self-contained application completely hosted on the
+Hub using Docker**. The diagram below illustrates the complete process.
+
+![ZenML on HuggingFace Spaces -- default deployment](hf_spaces_chart.png)
+
+Visit [the ZenML documentation](https://docs.zenml.io/) to learn more about its
+features and how to get started with running your machine learning pipelines
+through your Huggingface Spaces deployment. You can check out [some small sample
+examples](https://github.com/zenml-io/zenml/tree/main/examples) of ZenML pipelines to get started or take your pick of some more
+complex production-grade projects at [the ZenML Projects
+repository](https://github.com/zenml-io/zenml-projects). ZenML integrates with
+many of your favorite tools out of the box, [including
+Huggingface](https://zenml.io/integrations/huggingface) of course! If there's
+something else you want to use, we're built to be extensible and you can easily
+make it work with whatever your custom tool or workflow is.
+
+## ⚡️ Deploy ZenML on Spaces
+
+You can deploy ZenML on Spaces with just a few clicks:
+
+<a  href="https://huggingface.co/new-space?template=zenml/zenml-template-space">
+    <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg" />
+</a>
+
+To set up your ZenML app, you need to specify three main components: the Owner
+(either your personal account or an organization), a Space name, and the
+Visibility (a bit lower down the page). Note that the space visibility needs to
+be set to 'Public' if you wish to connect to the ZenML server from your local
+machine.
+
+PHOTO GOES HERE
```

Comment:
Which photo :-)

Reply:
We need our space to become an official space first, so we can take a photo that shows it as a clickable option. They said this would be no problem, but we need that to happen first.

Reply:
So should we take it out until that happens?

Reply:
I think their space will be created before this gets merged tbh. Once I get the go-ahead from @schustmi on the `Dockerfile` I'll make a PR and let them know we'd like them to set us up as official. It'll be handled.